File: cpp_e9zMZ/cpp/06-thaphinh/21.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    int height = 3;
    string shape = "";

    for (int i = 1; i <= height; i++) {
        string space = myRepeat(" ", height - i);
        string number = "";
        string record = "";

        for (int k = i; k >= 1; k--) {
            number += to_string(k);
        }
        
        record = space + number;
        shape += record + "\n";
    }

    cout << shape << endl;
}

File: python_ToSXs/python/06-thaphinh/21.py
height = 5

shape = ""
record = ""

for i in range(1, height + 1):
    space = " " * (height - i)

    character = [str(j) for j in range(i, 0, -1)]
    character = "".join(character)

    record = space + character
    shape += record + "\n"

print(shape)
